src/Validation.ContentScan.Core/IContentScanEnqueuer.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace NuGet.Jobs.Validation.ContentScan
+{
+    public interface IContentScanEnqueuer
+    {
+        /// <summary>
+        /// Enqueues Scan operation. The message delivery is going to be delayed by <see cref="ScanAndSignEnqueuerConfiguration.MessageDelay"/> setting.
+        /// </summary>
+        /// <param name="validationId">Validation ID for which scan is requested.</param>
+        /// <param name="pkgUrl">Url of the package to scan.</param>
+        Task EnqueueContentScanAsync(Guid validationId, string pkgUrl);
```

Comment:
Can we accept a `Uri` here instead of a `string`? Also, should we make the Uri parameter's name generic? I don't feel strongly about the parameter name, feel free to keep it as-is if you prefer that.

```suggestion
        Task EnqueueContentScanAsync(Guid validationId, Uri contentUri);
```

Reply:
done. accepting URI now